src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+
+const promise1 = new Promise(resolver1);
+
+function resolver1(complete, eror) {
```

Comment:
```suggestion
function callback1(resolve, reject) {
```
actually, the function (callback) that you passed to the new Promise has 2 arguments: resolve function and reject function (resolve and reject are default wide known names)
So to avoid confusion better to rename `resolver1`

Reply:
the same with resolver2